pkgs/development/interpreters/php/default.nix

```diff
@@ -227,7 +227,7 @@ in {
   # We can't update the darwin versions because they simply don't compile at
   # all due to a bug in the intl extensions.
   #
-  # The bug so far is present in 7.1.21, 7.1.22, 7.1.23, 7.2.9, 7.2.10, 7.2.11.
+  # The bug so far is present in 7.1.21, 7.1.22, 7.1.23, 7.2.9, 7.2.10, 7.2.12.
```

Comment:
@ryantm This change was probably not intended :smile: 

But from the changelog of 7.2.12 I haven't seen that they would have fixed the bug with darwin and the upstream issue isn't closed. So we can *probably* just add 7.2.12 to this list.

I haven't tested building 7.2.12 on darwin though.

Reply:
Sorry, nixpkgs-update is kind of dumb about updating version strings. I don't think I can make it much better until I parse nixpkgs to find the version location.